test/e2e/view-overview-om.js

```diff
@@ -30,7 +30,8 @@ describe('Offender Manager', function () {
 
       let capacity = await $('.sln-capacity')
       capacity = await capacity.getText()
-      expect(capacity).to.equal(`${workloadOwnerCapacity}%`)
+      const capacityPercent = capacity.substring(0, capacity.indexOf('%'))
```

Comment:
Needed to extract the text before the percent, otherwise the hidden text would be included